lib/api.js

```diff
@@ -73,7 +73,7 @@ function deleteHooks(reponame, url, token, callback) {
     .get(apiUrl)
     .set('Authorization', 'token ' + token)
     .set('User-Agent', 'StriderCD (http://stridercd.com)')
-    .end(function (res) {
+    .end(function (res) { //CHECK: have a callback with the first param as err and process that err
```

Comment:
thanks, roger that, I have already updated the pull request with this.

On Mon, Aug 17, 2015 at 9:30 PM, Ilya Radchenko <notifications@github.com>
wrote:

> In lib/api.js
> <https://github.com/Strider-CD/strider-github/pull/49#discussion_r37182520>
> :
>
> > @@ -73,7 +73,7 @@ function deleteHooks(reponame, url, token, callback) {
> >      .get(apiUrl)
> >      .set('Authorization', 'token ' + token)
> >      .set('User-Agent', 'StriderCD (http://stridercd.com)')
> > -    .end(function (res) {
> > +    .end(function (res) { //CHECK: have a callback with the first param as err and process that err
>
> function (err, res) is standard node callback pattern, so I think stick
> with that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Strider-CD/strider-github/pull/49/files#r37182520>.
>